students/Thai_H/lesson2/fizzbuzz.py

```diff
@@ -0,0 +1,12 @@
+for n in range(1, 101):
+    if n % 3 == 0:
+        print('Fizz')
+    elif n % 5 == 0:
+        print('Buzz')
+    elif n % 3 == 0 and n % 5 == 0:
+        print('FizzBuzz')
+    else:
+        print(n)
```

Comment:
Not getting any FizzBuzzes here - take a look at a simple way to rearrange this to get the desired result.